Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/firebase/AuthController.kt

```diff
@@ -44,13 +55,25 @@ class AuthController {
         return if (resultCode == Activity.RESULT_OK) {
             user = FirebaseAuth.getInstance().currentUser
             user?.let { user ->
-                FirebaseHelper.instance.userDatabase.child(user.uid)
-                    .setValue(
-                        createLocalUser(
-                            UserAgendaRepo.getInstance(context).savedSessionIds,
-                            user
-                        )
-                    )
+                FirebaseHelper.instance.userDatabase.addListenerForSingleValueEvent(object:
+                    ValueEventListener {
+                    override fun onCancelled(error: DatabaseError) {
```

Comment:
Not a huge deal, but if we edit this file I think this formatting is a little off? Maybe that's how the IDE wants it though.

Reply:
Weird...I tried to reformat it and it didn't change anything...

Reply:
I can see why the IDE would do it but the override inline with the class name kinda threw me off lol